### ISSUE TITLE
Not redirected to windowed login

### ISSUE THREAD

**Issue by lehsaclarkson** [RTWA/App-Staff_Directory#53]
When viewing a view as an unauthed user, you are not redirected to the windowed login, you get the main login which doesn't work in iframes

**Comment by RobTrehy** [comment]
I have just published Staff Directory App version 1.1.3 which fixes this

**Comment by lehsaclarkson** [comment]
Has 1.1.3 been published as I'm only seeing 1.1.2. 

Thank you

**Comment by RobTrehy** [comment]
I didn't press the button.
Just published it now - WebApps may have a cache though. So try Settings > Clear System Cache if it doesn't appear.

**Comment by lehsaclarkson** [comment]
I have no updated to 1.1.3 but it's causing the Staff Directory to now loop the login. 
I'm using the view created version (apps/StaffDirectory/view/nMJaI1ZEM7)

**Comment by RobTrehy** [comment]
Obviously, I can't authenticate. But it loaded the popup window for me?
Clear browser cache etc.

**Comment by lehsaclarkson** [comment]
I've cleared the cache and I have tried again. I have sent over some screenshots in an email.
Is there anyway to revert back to the previous version.

**Comment by lehsaclarkson** [comment]
Ok. Done. Thank you.

Also we have lost the scrollbars on the directory.

Mr A Clarkson
Applications Delivery Manager - IT Services

Lady Eleanor Holles
Hanworth Road, Hampton, Middlesex TW12 3HF
Telephone: 020 8979 1601
Twitter: @***@***.***>
Web: www.lehs.org.uk<http://www.lehs.org.uk/>

[Lady Eleanor Holles_RGB]
